models/usershopmapping.js

```diff
@@ -0,0 +1,16 @@
+const Joi = require('joi');
+const mongoose = require('mongoose');
+ 
+const UserShopmap = mongoose.model('UserShopmap', new mongoose.Schema({
```

Comment:
Please name this to some generic also file name should be user_shop_mapping and class name be UserShopMapping